File: Extra - Estudos/GithubNivelamento/ex1/main.c
#include <stdio.h>
#include <stdbool.h>

int main(){
    int array[100], size, number;
    
    scanf("%d", &size);

    for(int i = 0; i < size ; i++){
        scanf("%d", &array[i]);
    }

    bool found = false;

    printf("insira um numero?");
    scanf("%d", &number);

    for(int i = 0 ; i < size ; i++){
        if(array[i] == number){
        found = true;
        break;
    } 
}
    if(found){
        printf("Numero encontrado.\n");
    } else {
        printf("Numero nao encontrado.\n");
    }

    return 0;
}

File: Extra - Estudos/GithubNivelamento/ex1/Main.java
import java.util.*;

public class Main{

    public static boolean FindNumber(int size , int[] arr, int num){
            for(int i = 0 ; i < size; i++){
                if(arr[i] == num){
                    return true;
                }
            }
            return false;
        }
    
        public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
    
        System.out.println ("Digite o tamanho: ");
        int size = sc.nextInt();
    
        System.out.println("Digite o numero a ser encontrado");
        int num = sc.nextInt();
    
        int[] arr = new int[size];
    
        for(int i = 0 ; i < size ; i++ ){
            arr[i] = sc.nextInt();
        }
    
        boolean found = FindNumber(size, arr, num);

        if(found == true){
            System.out.println("sim");
        } else {
            System.out.println("nao");
        }
    }
}
